File: inf1000/uke1/Raad.java
/**
Oppgave 1.15: Har du raad?
*/

import java.util.Scanner;

class Raad {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        System.out.print("Skriv inn saldo: ");
        double saldo = Double.parseDouble(in.next());

        System.out.print("Hva koster varen?: ");
        double varepris = Double.parseDouble(in.next());

        if (varepris > saldo) {
            System.out.println("Du har ikke raad.");
        } else {
            System.out.println("Du har raad.");
        }
    }
}

File: inf1001/uke2/raad.py
# Oppgave 2.13: Har du raad?

saldo_inn = input("Tast inn saldo: ")
saldo = float(saldo_inn)

varepris_inn = input("Hvor mye koster varen?: ")
varepris = float(varepris_inn)

if (saldo >= varepris):
    print "Du har raad"
else:
    print "Du har ikke raad"
